pubspec.yaml

```diff
@@ -7,14 +7,14 @@ environment:
 
 dependencies:
   collection: ^1.14.11
-  analyzer: '>=0.40.0 <2.0.0'
+  analyzer: ^1.7.2
```

Comment:
Now that we don't support these other analyzer versions, can we remove these lines from this matrix that test on older, now-unsupported analyzer versions?  https://github.com/Workiva/over_react/blob/master/.github/workflows/dart_ci.yml#L89-L91

Even though that build's redundant, you can just remove those lines and leave the matrix in place if you want, since it'll come in handy if we need to support multiple major versions.

Reply:
Actually, our Dart CI needs some cleanup to disable the failing stable build, which I can address in a separate PR.

That touches some of these lines, so I can just do that in that PR; don't worry about it for this PR.